src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -2,6 +2,7 @@ import { PfColors } from '../../../components/Pf/PfColors';
 import { EdgeLabelMode } from '../../../types/GraphFilter';
 import { config } from '../../../config';
 import { FAILURE, DEGRADED } from '../../../utils/Health';
+import { GraphType, CytoscapeGlobalScratchNamespace, CytoscapeGlobalScratchData } from '../../../types/Graph';
```

Comment:
not related to this PR, but I hate these relative `../../../../../../paths/to/files`